jwst/ramp_fitting/ramp_fit_step.py

```diff
@@ -414,8 +416,16 @@ def process(self, input):
             readnoise_filename = self.get_reference_file(input_model, 'readnoise')
             gain_filename = self.get_reference_file(input_model, 'gain')
 
-            log.info('Using READNOISE reference file: %s', readnoise_filename)
-            log.info('Using GAIN reference file: %s', gain_filename)
+            ngroups = input_model.data.shape[1]
+            if self.algorithm.upper() == "LIKELY" and ngroups < LIKELY_MIN_NGROUPS:
+                log.info(f"When selecting the LIKELY ramp fitting algorithm the"
+                          " ngroups needs to be a minimum of {likely_fit.LIKELY_MIN_NGROUPS},"
+                          " but ngroups = {ngroups}.  Due to this, the ramp fitting algorithm"
+                          " is being changed to OLS_C")
```

Comment:
This message is long - let's split it up into two messages and make them warnings.  Also, the f needs to go at the front of all the substrings to get the variables in there properly.

```suggestion
                log.warning(f"When selecting the LIKELY ramp fitting algorithm the"
                            f" ngroups needs to be a minimum of {LIKELY_MIN_NGROUPS},"
                            f" but ngroups = {ngroups}.")
                log.warning("Due to this, the ramp fitting algorithm"
                            " is being changed to OLS_C.")
```

Reply:
Thanks for fixing the variable reference.  Can you please also add the `f` at the beginning of the substrings, so the variables are printed properly?  Also, split into two messages and make it a warning instead of an info.